lib/client.js

```diff
@@ -12,17 +12,23 @@ var _reactRouterDom = require("react-router-dom");
 
 var _reduxSaga = _interopRequireDefault(require("redux-saga"));
```

Comment:
I'll need to take a closer look at this when I have more time, there might be an easier way